src/backend/src/common/enums/delegated-review-types.enum.ts

```diff
@@ -0,0 +1,4 @@
+export enum DelegatedReviewTypesEnum {
+  NON_PIA = 'NON-PIA',
+  CHECKLIST = 'CHECKLIST',
+}
```

Comment:
This is turning into checkbox boolean values as per the designs.

Reply:
This looks like a checkbox in designs - but I assume the behaviour should just be one out of many - like radio. I don't think you could select Non-PI and Checklist at the same time? 🤔 Same for Standard PIA vs Initiative Update vs Delegate Review.

<img width="814" alt="image" src="https://user-images.githubusercontent.com/87394256/200924610-860e1141-28d9-4e36-a0f9-7d017dac64ba.png">

Reply:
Guess you could be right about "What kind of delegated review is it?" question. However for "What kind of PIA are you submitting?" we can only select one.

<img width="967" alt="image" src="https://user-images.githubusercontent.com/87394256/200926480-257a23d0-56ae-4566-9cfc-06bbed019ef1.png">